plover/gui_qt/dictionary_editor.py

```diff
@@ -19,10 +19,11 @@
 
 from plover.translation import escape_translation, unescape_translation
 from plover.misc import expand_path, shorten_path
-from plover.steno import normalize_steno
+from plover.steno import normalize_steno, compare_normalize_stroke
 
 from plover.gui_qt.dictionary_editor_ui import Ui_DictionaryEditor
 from plover.gui_qt.utils import ToolBar, WindowState
+from functools import cmp_to_key
```

Comment:
https://www.python.org/dev/peps/pep-0008/#imports

In Python, system imports go on top of local imports according to PEP8. To quote:


> 1. standard library imports
> 2. related third party imports
> 3. local application/library specific imports